src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine bulldozerOne = new Bulldozer();
+        Machine excavatorOne = new Excavator();
```

Comment:
why did you name it bulldozerOne and excavatorOne instead of just bulldozer and excavator?

Reply:
Okay, thanks.